documentation/api/guides/index.md

```diff
@@ -1,5 +1,5 @@
 # Guides
 
-```{include} ../../_toc_templates/flat.md
```

Comment:
This isn't directly relevant but why do we use `_toc_templates` and not define the `toctree` manually to list content in a logical order?

Reply:
We certainly could, I just found it tedious for sections where there was no clear reason to order things in any particular way. People frequently forget to add pages to the toctree directives[^note], so I think globs are nice in that way. Repeating the `:titlesonly:` and other options was tedious to me, so I tried originally to use these include directives, but they're not properly flexible.

Anyway, I'm not convinced that the approach in this PR is better than doing as you said, just writing out the toctrees individually. I just found it really tedious to repeat all the options for the two common "types" of toctrees we use.

[^note]: Seriously, I've had to fix so many "missing from toctree" warnings.